static/js/public/snap-details/exapandable-distr-chart.js

```diff
@@ -0,0 +1,29 @@
+export default function initExpandableDistroChart() {
+  const moreEl = document.querySelector(".snapcraft-distro-chart__more");
+  const lessEl = document.querySelector(".snapcraft-distro-chart__less");
+  const detailsEl = document.querySelector(".snapcraft-distro-chart");
+
+  if (moreEl && lessEl && detailsEl && screen.width <= 772) {
+    moreEl.classList.remove("u-hide");
```

Comment:
Doing the above would mean we don't need to fiddle with hiding and showing via js.